aws/internal/keyvaluetags/generators/listtags/main.go

```diff
@@ -178,11 +181,20 @@ func {{ . | Title }}ListTags(conn {{ . | ClientType }}, identifier string{{ if .
 		return New(nil), err
 	}
 
-	return {{ . | Title }}KeyValueTags(output.{{ . | ListTagsOutputTagsField }}), nil
+	return {{ . | KeyValueTagsPrefix }}KeyValueTags(output.{{ . | ListTagsOutputTagsField }}), nil
```

Comment:
Instead of creating more customization in this generator, which would apply to any other weird packages in the AWS Go SDK using cross-package types, I think it might be nice to instead support `WafregionalKeyValueTags()` from the `servicetags` generator just like any other service. 👍 

To support that we can introduce a new customization function such as:

```go
// ServiceTagPackage determines the service tagging tag type value field.
func ServiceTagPackage(serviceName string) string {
	switch serviceName {
	case "wafregional":
		return "waf"
	default:
		return serviceName
	}
}
```

And replace the following lines:

```
func (tags KeyValueTags) {{ . | Title }}Tags() []*{{ . }}.{{ . | TagType }} {

func {{ . | Title }}KeyValueTags(tags []*{{ . }}.{{ . | TagType }}) KeyValueTags {
```

With the following:

```
func (tags KeyValueTags) {{ . | Title }}Tags() []*{{ . | TagPackage }}.{{ . | TagType }} {

func {{ . | Title }}KeyValueTags(tags []*{{ . | TagPackage }}.{{ . | TagType }}) KeyValueTags {
```